Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -671,6 +672,9 @@ import getElement from '../getElement.js';
      * removing the widget from layout.
      */
     CesiumWidget.prototype.destroy = function() {
+        if (this._scene) {
+            this._scene.renderError.removeEventListener(this._onRenderError);
```

Comment:
Since we are destroying the scene on the next line, why do we need to unsubscribe the event listener?  Destroying the scene should ensure that `renderError` and any subscriptions get cleaned up.  Do you have an example that shows that's not happening?

Reply:
Sorry for the slow roll here, I somehow missed this response.  While I'd still like to understand the circular dependency causing the GC not to clean up the memory, no reason we can't err on the side of caution here and properly clean up after ourselves.  I merged in master and updated CHANGES.  I'll merge as soon as CI is happy.  Thanks @jony89

Reply:
It's less of a circular dependency more like closures and references that stay alive while the anonymous function still have reference. I've discovered it by using heap snapshots and their deltas, one snapshot before destroying the viewer and one afterwards.
Thanks @mramato .